libs/tools/generator/core/src/engine/settings.ts

```diff
@@ -0,0 +1,20 @@
+/** Api configuration for forwarders that support custom domains. */
+export type EmailDomainSettings = {
+  /** The domain part of the generated email address.
+   *  @remarks The domain should be authorized by the forwarder before
+   *           submitting a request through bitwarden.
+   *  @example If the domain is `domain.io` and the generated username
+   *  is `jd`, then the generated email address will be `jd@mydomain.io`
```

Comment:
⛏️  Should be `domain.io` instead of `mydomain.io`
```suggestion
   *  is `jd`, then the generated email address will be `jd@domain.io`
```

Reply:
Addressed in c8bfa95f41662.